app/Http/Controllers/ReportController.php

```diff
@@ -35,7 +35,7 @@ public function store_game_report(Request $request, $gameid){
         return redirect()->action('ReportController@index_user', [\Auth::id()]);
     }
 
-    public function index_user($userid){
+    public function index_user($userid) {
```

Comment:
__[code_sniffer]__
* Method name "ReportController::index_user" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->